org.eclipse.buildship.kotlinDSL.provider/build.gradle

```diff
@@ -0,0 +1,7 @@
+plugins {
+    id 'application'
+}
+apply plugin: eclipsebuild.BundlePlugin
+repositories {
```

Comment:
❓ Why do you need to declare a repository if you don't declare any dependencies?